packages/server/tests/helpers/utils.ts

```diff
@@ -150,4 +152,19 @@ export class Utils {
     }
     return res.toString();
   };
+
+  static ethCall = async (
```

Comment:
Update name to make clear there's a retry e.g ethCallWRetries or something.
We want to make sure people use this method only when they want retries

Reply:
Done.